app/questionnaire/questionnaire_manager.py

```diff
@@ -56,7 +56,7 @@ def validate_all_answers(self):
 
         return True, None
 
-    def update_questionnaire_store(self, location):
+    def update_questionnaire_store(self, location, save=True):
```

Comment:
Internet reference from a [great book](https://www.amazon.co.uk/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882)
Flag arguments are ugly. Passing a boolean into a function is a truly terrible practice. It immediately complicates the signature of the method, loudly proclaiming that this function does more than one thing. It does one thing if the flag is true and another if the flag is false!